Workbooks/WorkbooksMetadata.json

```diff
@@ -1113,6 +1113,22 @@
     "title": "IntSights IOC Workbook",
     "templateRelativePath": "IntsightsIOCWorkbook.json", 
     "subtitle": "",
+    "provider": "Intsights Cyber Intelligence"
+  },
+  {
+ "workbookKey": "ForcepointCloudSecurityGatewayWorkbook",
+ "logoFileName": "Forcepoint_new_logo.svg",
+ "description": "Use this report to understand query runs across your workspace.",
+ "dataTypesDependencies": ["CommonSecurityLog"],
+ "dataConnectorsDependencies": ["ForcepointCloudSecurityGateway"],
+ "previewImagesFileNames": ["ForcepointCloudSecurityGatewayWhite.PNG","ForcepointCloudSecurityGatewayBlack.PNG"],
+ "version": "1.0",
+ "title": "Forcepoint Cloud Security Gateway Workbook",
+ "templateRelativePath": "ForcepointCloudSecuirtyGatewayworkbook.json",
+ "subtitle": "",
+ "provider": "Forcepoint"
+ }
+=======
```

Comment:
Please resolve conflicts properly,[Remove duplicate code of line no 1131,1132,1133 and update '**,**' at line no 1130]